let $x := fn:error()
return $x
